config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  get "homepages/index"
```

Comment:
Interesting, I'm not sure what this syntax does... Do we need it? A lot of the tests break if I comment this out... but maybe at those points you should use `root_path`?